dotnet/src/SemanticKernel.Abstractions/Orchestration/PlanningException.cs

```diff
@@ -3,7 +3,7 @@
 using System;
 using Microsoft.SemanticKernel.Diagnostics;
 
-namespace Microsoft.SemanticKernel.Planning;
+namespace Microsoft.SemanticKernel.Orchestration;
 
 /// <summary>
```

Comment:
I'd expect this to stay in the package it's used in and the `planning` namespace.

Reply:
I'm going back to previous design conversations here: plans are functions compositions, so `Plan` and `PlanningException` are in the same category of ISKfunction, ContextVariables, etc.